docs/website/content/v0.3/en/guides/cloud/vmware.md

```diff
@@ -0,0 +1,167 @@
+---
+title: 'VMware'
+---
+
+## Creating a Cluster via the govc CLI
+
+In this guide we will create an HA Kubernetes cluster with 3 worker nodes.
+We will use the `govc` cli which can be downloaded [here](https://github.com/vmware/govmomi/tree/master/govc#installation).
+
+### Prereqs
+
+Prior to starting, it is important to have the following infrastructure in place and available:
+
+- DHCP server
+- Load Balancer or DNS address for cluster endpoint
+  - If using a load balancer, the most common setup is to balance `tcp/443` across the control plane nodes `tcp/6443`
+  - If using a DNS address, the A record should return back the addresses of the control plane nodes
+
+### Create the Machine Configuration Files
+
+#### Generating Base Configurations
+
+Using the DNS name or name of the loadbalancer used in the prereq steps, generate the base configuration files for the Talos machines:
+
+```bash
+$ osctl config generate talos-k8s-vmware-tutorial https://$LB_DNS_NAME
+created init.yaml
+created controlplane.yaml
+created join.yaml
+created talosconfig
+```
+
+At this point, you can modify the generated configs to your liking.
+
+#### Validate the Configuration Files
+
+```bash
+$ osctl validate --config init.yaml --mode cloud
+init.yaml is valid for cloud mode
+$ osctl validate --config controlplane.yaml --mode cloud
+controlplane.yaml is valid for cloud mode
+$ osctl validate --config join.yaml --mode cloud
+join.yaml is valid for cloud mode
+```
+
+### Set up environment variables
+
+`govc` makes use of the following environment variables
+
+```bash
+export GOVC_URL=https://myvcenter
+export GOVC_USERNAME=root
+export GOVC_PASSWORD=Whywas6afraidof7,because789
+# If your vCenter installation makes use of self signed certificates,
+# you'll want to have GOVC_INSECURE=true set
+# export GOVC_INSECURE=true
+```
+
+### Download the OVA
+
+A talos.ova asset is published with each release:
+
+```bash
+curl -LO https://github.com/talos-systems/talos/releases/download/v0.3.0-alpha.2/talos.ova
+```
+
+### Import the OVA into vCenter
+
+We'll need to repeat this step for each Talos node we want to create.
+In a typical HA setup, we'll have 3 control plane nodes and N workers.
+In the following example, we'll set up a HA control plane with two worker nodes.
+
+```bash
+govc import.ova -name control-1 $talos/build/talos.ova
+govc import.ova -name control-2 $talos/build/talos.ova
+govc import.ova -name control-3 $talos/build/talos.ova
+govc import.ova -name worker-1 $talos/build/talos.ova
+govc import.ova -name worker-2 $talos/build/talos.ova
+```
+
+#### Update Settings for the Bootstrap Node
+
+Talos makes use of the `guestinfo` facility of VMware to provide the machine/cluster configuration.
+This can be set using the `govc vm.change` command.
+`disk.enableUUID=1` is used to facilitate persistent storage using the vSphere cloud provider integration with Kubernetes.
```

Comment:
nit: Starting with code makes the sentence ugly. What if we flipped it like:

```
To facilitate persistent storage using the vSphere cloud provider integration with Kubernetes, `disk.enableUUID=1` is used.
```

Or something like that.